### ISSUE TITLE
Trennung zwischen JUG-Terminen und sonstigen

### ISSUE THREAD

**Issue by meistermeier** [JUG-Ostfalen/JUG-Ostfalen.github.io#31]
JUG-Termine sollten evtl. etwas mehr hervorgehoben werden als sonstige. An der Stelle müssen wir schauen, was wir mit dem Hacktalk machen, da dieser irgendwie für mich "in-between" steht.

**Comment by schauder** [comment]
Für mich ist der Hacktalk 3rd Party. Auch wenn der Hacktalk mir persönlich mehr am Herzen liegt als die Konferenz oder dergleichen, nehmen die meisten JUG Mitglieder diese Events zwar zur Kenntnis gehen aber nur selten hin.

**Comment by meistermeier** [comment]
Grundsätzlich würde ich da im ersten Wurf einfach mit der Hintergrundfarbe der Termine etwas spielen wollen. Muss ja kein Mega-Make-Over werden, wenn man es schon so etwas erkennen kann.

**Comment by schauder** [comment]
Ja, sehe ich ähnlich. Und bei der Gelegenheit vielleicht dir Reihenfolge zw. Artikeln und Events vertauschen, damit letztere auf Mobilen Geräten als erstes angezeigt werden.
